app/javascript/components/App.js

```diff
@@ -22,16 +21,83 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      searches: mockSearch,
+      searches: [],
     };
   }
 
+  componentDidMount() {
+    this.getSearches();
+  }
+
+  getSearches = () => {
+    fetch("/ticity_searches")
+      .then((response) => {
+        return response.json();
+      })
```

Comment:
Just FYI, you can get rid of curly braces  and `return` keyword and just put response.json() after the fat arrow.